.github/workflows/test_docs_build.yml

```
@@ -0,0 +1,56 @@
+name: Documentation
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+      - develop
+  schedule:
+    # Run every day at 3 am UTC
+    - cron: "0 3 * * *"
+
+env:
+  PYBAMM_DISABLE_TELEMETRY: "true"
+  FORCE_COLOR: 3
+  PYBAMM_IDAKLU_EXPR_CASADI: OFF
+  PYBAMM_IDAKLU_EXPR_IREE: OFF
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Install Linux system dependencies
+        uses: awalsh128/cache-apt-pkgs-action@a6c3917cc929dd0345bfb2d3feaf9101823370ad # v1.4.2
+        with:
+          packages: graphviz pandoc
+          execute_install_scripts: true
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+          cache: 'pip'
+
+      - name: Install PyBaMM with [docs,dev,all]
+        run: |
+          python -m pip install --upgrade pip
+          pip install -e .[docs,dev,all]
+
+      - name: Run doctests
+        run: python -m nox -s doctests
+
+      - name: Build docs (HTML)
+        run: sphinx-build -b html docs/ docs/_build/html
```

It looks like this was missed from @agriyakhetarpal's review
```suggestion
      - name: Build docs (HTML)
        run: sphinx-build -b html -d docs/_build/.doctrees docs docs/_build
```

---

It looks like this is still not changed when I view the review. I will wait for @agriyakhetarpal to do his last review